### PR TITLE
Fix logs and add customer_email_log

### PR DESCRIPTION
**Pull request by kiragrom**

### CODE REVIEW

File: lib/send_with_us/api.rb [hunk 1]
```diff
@@ -277,10 +277,24 @@ def customer_remove_from_group(email_address, group_id)
...
-    def logs(options = {})
-      endpoint = "logs"
+    def customer_email_log(email_address, options = {})
+      endpoint = "customers/#{email_address}/logs"
+      params   = {}
+
+      params[:count]       = options[:count]       unless options[:count].nil?
+      params[:created_gt]  = options[:created_gt]  unless options[:created_gt].nil?
+      params[:created_lt]  = options[:created_lt]  unless options[:created_lt].nil?
+      if params.present?
```
Comment (moxuz) [inline lib/send_with_us/api.rb]: @kiragrom this looks like a RoR thing, which means we can't use it here. It's causing our Travis build to fail for your PR (https://travis-ci.org/sendwithus/sendwithus_ruby/builds/116131699)
If you can get this working without .present? and our build passes it looks good!

File: lib/send_with_us/api.rb [hunk 2]
```diff
@@ -289,9 +303,9 @@ def logs(options = {})
       params[:created_lt]  = options[:created_lt]  unless options[:created_lt].nil?
       params[:created_lte] = options[:created_lte] unless options[:created_lte].nil?
 
-      unless params
-        params = URI.encode_www_form(params)
-        endpoint = endpoint + "?" + params
+      if params.present?
```
Comment (moxuz) [inline lib/send_with_us/api.rb]: @kiragrom same thing here.

**Comment by moxuz** [comment]
@kiragrom thanks for fixing with ".exist?". Would you be able to add a unit test for the new customer_email_logs functionality?

**Comment by kiragrom** [comment]
@moxuz Done.

**Comment by moxuz** [comment]
@kiragrom Thank you :smile_cat:

**Pull request merged**